crates/fiber/src/unix.rs

```diff
@@ -74,22 +74,18 @@ unsafe fn mmap_new_stack(size: usize, page_size: usize) -> io::Result<(*mut u8,
 
 /// Maps a new stack with a guard page. Returns (top, len).
 #[cfg(target_os = "openbsd")]
-unsafe fn mmap_new_stack(size: usize, page_size: usize) -> io::Result<(*mut u8, usize)> {
+unsafe fn mmap_new_stack(size: usize, _page_size: usize) -> io::Result<(*mut u8, usize)> {
     // On OpenBSD, we need to use MAP_STACK to specify that a page can contain a
     // stack; otherwise, we will get a SIGSEGV when it is used as such.
     //
-    // Note also that MAP_STACK must be specified with PROT_READ|PROT_WRITE, and
+    // Note that MAP_STACK must be specified with PROT_READ|PROT_WRITE, and
```

Comment:
`MAP_STACK` can also be used on Linux. It is currently a no-op, but according to the man page it is accepted such that if a future architecture requires special handling for the stack, no program changes are necessary.